addons/mail/static/src/model/model_manager.js

```diff
@@ -1055,7 +1057,7 @@ class ModelManager {
      * @param {Object} [options]
      * @returns {boolean} whether any value changed for the current record
      */
-    _update(record, data, options) {
+    _update(record, data, options = { allowWriteReadonly: false }) {
```

Comment:
This is not the right syntax for a default value, if we pass an object without the key, the key will not be present on it this way.

Either you spread the object (which we don't want here because it can be used to forward arbitrary options to fields), or the default value has to be handled on a new line.